ansible/prater.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Pre-playbook checks
+  hosts: localhost
+  tasks:
+    - name: 'Verify Ansible version is 2.8 or newer'
+      assert:
+        that: 'ansible_version.full is version_compare("2.8", ">=")'
+        msg: 'Your Ansible version is lower than 2.8. Upgrade it.'
+
+- name: Configure build nodes
+  become: true
+  hosts:
+    - testing-large-01.aws-eu-central-1a.nimbus.prater
+  roles:
+    - { role: beacon-node-builds,     tags: beacon-node-builds }
```

Comment:
Is there a point to this? If we're using the same size as large in Pyrmon - `z1d.large` - we could also use the `testing-large-01.aws-eu-central-1a.nimbus.pyrmont` host.

The image names and tags do not distinguish between testnets:
```
admin@testing-small-03.aws-eu-central-1a.nimbus.pyrmont:~ % d
CONTAINER ID   NAMES                               IMAGE                                         CREATED      STATUS
558bb9301fd8   beacon-node-pyrmont-testing-small   statusteam/nimbus_beacon_node:testing-small   2 days ago   Up 2 days
```
Just between branches and host size.